taskcluster/translations_taskgraph/actions/train.py

```diff
@@ -64,6 +64,7 @@ def validate_pretrained_models(params):
     symbol="train",
     description="Initiate part or all of the training pipeline",
     cb_name="train",
+    permission="train",
```

Comment:
Heh, that's a lot of "train"

Reply:
:train: :train2: :bullettrain_side: :steam_locomotive: :bullettrain_front: